CONTRIBUTING.md

```diff
@@ -151,6 +152,125 @@ Validate authentication using the username `alice` with the password `alice`:
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+
+
+#### Google Cloud Platform (GCP) Authentication
+
+To enable a host to log into Conjur using GCP identity token, run `start` with the `--authn-ldap` flag and pass a name of a running Google Compute Engine (GCE).
```

Comment:
To enable a host to log in to DAP/Conjur using a GCP identity token, run `start` with the `--authn-ldap` flag and pass the name of a running Google Compute Engine (GCE).



oh wait.. is this for OSS only? Then no DAP! Sorry